src/NHibernate.Test/NHibernate.Test.csproj

```diff
@@ -56,7 +56,7 @@
     <PackageReference Include="Microsoft.Data.SqlClient" Version="1.0.19269.1" />
     <PackageReference Include="System.Data.SQLite.Core" Version="1.0.109.2" />
     <PackageReference Include="System.Linq.Dynamic.Core" Version="1.0.8.11" />
-    <PackageReference Include="NSubstitute" Version="3.1.0" />
+    <PackageReference Include="NSubstitute" Version="4.2.1" />
```

Comment:
Upgraded in order to use `Configure` method, which allows to mock `Dialect.SupportsVariableLimit` without also mocking `Dialect.SupportsLimit`.